nixos/modules/services/web-apps/baget.nix

```diff
@@ -1,170 +1,16 @@
-{ config, lib, pkgs, ... }:
+{ lib, ... }:
 
 with lib;
 
 let
-
-  cfg = config.services.baget;
-
-  defaultConfig = {
-    "PackageDeletionBehavior" = "Unlist";
-    "AllowPackageOverwrites" = false;
-
-    "Database" = {
-      "Type" = "Sqlite";
-      "ConnectionString" = "Data Source=baget.db";
-    };
-
-    "Storage" = {
-      "Type" = "FileSystem";
-      "Path" = "";
-    };
-
-    "Search" = {
-      "Type" = "Database";
-    };
-
-    "Mirror" = {
-      "Enabled" = false;
-      "PackageSource" = "https://api.nuget.org/v3/index.json";
-    };
-
-    "Logging" = {
-      "IncludeScopes" = false;
-      "Debug" = {
-        "LogLevel" = {
-          "Default" = "Warning";
-        };
-      };
-      "Console" = {
-        "LogLevel" = {
-          "Microsoft.Hosting.Lifetime" = "Information";
-          "Default" = "Warning";
-        };
-      };
-    };
-  };
-
-  configAttrs = recursiveUpdate defaultConfig cfg.extraConfig;
-
-  configFormat = pkgs.formats.json {};
-  configFile = configFormat.generate "appsettings.json" configAttrs;
+  removed = k: lib.mkRemovedOptionModule [ "config" "services" "baget" k ]
+    "The baget module was removed due to the upstream package being unmaintained.";
```

Comment:
done, however the build still fails since the nixos manual can't be build - `services.baget.enabled` is referenced in the changelog for old NixOS version where it was added. Do I also remove this reference?

Reply:
I removed the reference and that fixed manual build, let me know if that's ok